src/App.tsx

```diff
@@ -1,9 +1,18 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
+import { Movie } from './react-app-env';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies: Movie[] = moviesFromServer.filter((movie) => {
+    const titleAndDescription = `${movie.title} ${movie.description}`;
+
+    return titleAndDescription.toLowerCase().includes(query.toLowerCase());
```

Comment:
Now you set query to lower case in each callback call
Will be better to create separate variable before the filter method where you can store query in lower case and use it here